pkg/collector/corechecks/gpu/gpu.go

```diff
@@ -155,12 +161,22 @@ func (m *Check) emitSysprobeMetrics(snd sender.Sender) error {
 	return nil
 }
 
-func getTagsForKey(key model.StatsKey) []string {
-	// Per-PID metrics are subject to change due to high cardinality
-	return []string{
+func (m *Check) getTagsForKey(key model.StatsKey) []string {
+	entityID := taggertypes.NewEntityID(taggertypes.ContainerID, key.ContainerID)
+	tags, err := m.tagger.Tag(entityID, m.tagger.ChecksCardinality())
```

Comment:
I think for localTagger this will [return](https://github.com/DataDog/datadog-agent/blob/cedfd68a55bc86be6edefb391a124ae9791f1d63/comp/core/tagger/impl/local_tagger.go#L173) lowCardinality, while looking for other [references](https://github.com/DataDog/datadog-agent/blob/482b8e0624b108dec97e83b311489ba9874ef9c7/pkg/collector/corechecks/containers/generic/processor.go#L78) that use `taggerTypes.ContainerID` it seems that `HighCardinality` should be used

Reply:
can you check with the containers team what is the best practice?